test/unit/src/lib/utils/firebase-loader.js

```diff
@@ -0,0 +1,29 @@
+const {assert} = require('../assert');
+const loader = require('../../../../../src/lib/utils/firebase-loader');
+
+describe('firebase-loader', function() {
+  describe('loader', function() {
```

Comment:
Can you write a test to assert that on the initial call it returns a function and another test to verify that calling that returned function returns a promise. Since we're not using typescript compiler yet I think the tests would be the best way for us to enforce the contract.

Reply:
I've done this.